gr-uhd/examples/grc/uhd_msg_tune.grc

```diff
@@ -142,36 +133,132 @@ blocks:
     bus_sink: false
     bus_source: false
     bus_structure: null
-    coordinate: [144, 307]
+    coordinate: [144, 244.0]
     rotation: 0
     state: enabled
-- name: samp_rate
-  id: variable
+- name: args
+  id: parameter
   parameters:
+    alias: ''
     comment: ''
-    value: 2e6
+    hide: none
+    label: Device Arggs
+    short_id: a
+    type: str
+    value: type=x300
```

Comment:
We could remove that, to remove one more obstacle for newbs, but honestly, if you attach a B200 and wonder why a sink configured for type=x300 can't connect to it…

Reply:
Good catch, removed. I also found a typo that I fixed.